src/gas/fetchGasEstimatesViaEthFeeHistory.ts

```diff
@@ -1,154 +1,9 @@
-// This code is translated from the MetaSwap API:
```

Comment:
Basically, a lot of these functions got extracted to separate files and so did their tests.